enhancements/fbc-run-bundle.md

```diff
@@ -0,0 +1,268 @@
+---
+title: fbc-run-bundle
```

Comment:
I would rename the file `sdk-fbc-run-bundle.md` so that we know it is an SDK enhancement since this repo has things from different projects.

```suggestion
title: sdk-fbc-run-bundle
```